talos/hail_filter_sv.py

```diff
@@ -136,20 +149,30 @@ def main(mt_path: str, panelapp_path: str, pedigree: str, vcf_out: str):
         pedigree ():
         vcf_out (str): where to write VCF out
     """
+    get_logger(__file__).info(
```

Comment:
Should define this somewhere central to reuse, given everything is already a script